http://bugs.gentoo.org/176558

ripped from suse's rarpd-1.1-631.src.rpm:
* Wed Apr 18 2007 - nadvornik@suse.cz
- fixed unlimited packet growth [#246891]

--- rarpd.c.orig      2007-02-21 11:41:25.099136000 +0100
+++ rarpd.c   2007-02-21 12:39:49.837404920 +0100
@@ -307,6 +307,8 @@ void send_rarpreply(unsigned char *dest_
     syslog(LOG_ERR, "%s: Write error: %s", program_name, libnet_geterror(ld));
     exit(1);
   }
+
+  libnet_clear_packet(ld);
 }
